scripts-dev/complement.sh

```diff
@@ -31,11 +32,12 @@ cd "$(dirname $0)/.."
 
 # Check for a user-specified Complement checkout
 if [[ -z "$COMPLEMENT_DIR" ]]; then
+  COMPLEMENT_REF=${COMPLEMENT_REF:-master}
   echo "COMPLEMENT_DIR not set. Fetching the latest Complement checkout..."
```

Comment:
We should probably clarify this log line too, what do you think of:

```suggestion
  echo "COMPLEMENT_DIR not set. Fetching Complement checkout from ${COMPLEMENT_REF}..."
```